.github/CODEOWNERS

```diff
@@ -0,0 +1,7 @@
+# Lines starting with '#' are comments.
+# Each line is a file pattern followed by one or more owners.
+
+*                    @protolambda @norswap @trianglesphere @maurelian @karlfloersch @ben-chain @cfromknecht @mslipper @tynes
+/l2os/               @protolambda @norswap @trianglesphere @cfromknecht @mslipper @tynes
+/opnode/             @protolambda @norswap @trianglesphere @cfromknecht @mslipper @tynes
+/packages/contracts/ @protolambda @norswap @maurelian @karlfloersch @smartcontracts 
```

Comment:
This will automatically assign reviews accordingly right? Seems like it'll be super noisy.

Reply:
5+ reviewers per PR by default is indeed quite noisy

Reply:
We need to make sure that it is configured to select 1 or 2 reviewers (can be done somewhere in github), but yes, the intention is to automatically select reviewers.

Reply:
https://docs.github.com/en/organizations/organizing-members-into-teams/managing-code-review-settings-for-your-team

Reply:
IMO we should just choose one or two ppl who can actually _own_ each area of the repo. Either those people can review the pr, or will take responsibility for finding someone who can.